src/Makefile.am

```diff
@@ -314,14 +317,21 @@ libbitcoin_common_a_SOURCES = \
   keystore.cpp \
   netaddress.cpp \
   netbase.cpp \
-  policy/feerate.cpp \
   protocol.cpp \
   scheduler.cpp \
   script/sign.cpp \
   script/standard.cpp \
   warnings.cpp \
   $(BITCOIN_CORE_H)
 
+# policy: shared between all executables that require policy rules.
+libbitcoin_policy_a_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+libbitcoin_policy_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+libbitcoin_policy_a_SOURCES = \
```

Comment:
This is a bit confusing to me.

The sources list doesn't contain all of policy/*.cpp (`fees.cpp` and `inputs.cpp` could be in there?), and the ones listed below are not used together anywhere (i.e. `feerate.cpp` is used in `libbitcoin_common` which doesn't need `policy.cpp`, and `libbitcoin_server` relies on `policy.cpp` but not `feerate.cpp`).

Why is this shared library needed, and why this particular combination?